### ISSUE TITLE
question for the unsupervised extractive summarization method

### ISSUE THREAD

**Issue by changan112** [subhendukhatuya/FLAN-FinBPS#1]
Hello! I’m glad you’re interested in the unsupervised extractive summarization method. Here are my questions:
(1) I wonder whether this method is applicable to Chinese non-bulleted summarization? Also, would questions generated by a question generation model that has not been fine-tuned have a negative effect?
(2) Regarding the part about topic keywords, is it obtained by using the topic_modelling.py file on the question list (Question_keywords.txt) from the training set? Then, for each test set content, we search for which topic keywords are included, and obtain the questions corresponding to the topic keywords as the questions for the test set, and then calculate the similarity, right? But wouldn’t the test set include some topic keywords that were not included in the training set?
